pom.xml

```diff
@@ -25,6 +25,7 @@
         <module>elide-contrib</module>
         <module>elide-standalone</module>
         <module>elide-spring</module>
+        <module>elide-async</module>
```

Comment:
The list of modules specifies the order of the build.  elide-async should be moved higher up so we can use it in other places (like spring and standalone).